app/services/symbolService.js

```diff
@@ -96,8 +96,68 @@ class SymbolService {
     if (params.on_time !== undefined) processedSymbol.on_time = params.on_time;
     if (params.status !== undefined) processedSymbol.status = params.status;
 
-    const newExchangeSymbol = await new DataExchangeSymbolModel(processedSymbol).save();
-    return { newExchangeSymbol };
+    const newDataExchangeSymbol = await new DataExchangeSymbolModel(processedSymbol).save();
+    return { newDataExchangeSymbol };
+  }
+
+  async getPrice() {
+    const url = 'https://api.binance.com/api/v3/klines';
+    const SYMBOL = 'BTCUSDT';
+    const EXCHANGE = 'binance';
+
+    try {
+      const responseUSD = await axios.get(url, {
+        params: {
+          symbol: SYMBOL,
+          interval: '1d',
+          limit: 1 
+        }
+      });
+
+      const dataUSD = responseUSD.data[0];
+      const price_usd = parseFloat(dataUSD[4]); 
+      const volume_usd = parseFloat(dataUSD[5]);
+
+      const usdToCnyRate = await this.getUsdToCnyRate();
+      const price_cny = price_usd * usdToCnyRate;
+      const volume_cny = volume_usd * usdToCnyRate;
+
```

Comment:
Why is the camel hump method sometimes used to define variables, and sometimes underlined to separate them?